dts/arm/nxp/nxp_rt.dtsi

```diff
@@ -41,6 +41,22 @@
 			};
 		};
 
+		flexspi0: flexspi0@402a8000 {
+			compatible = "nxp,imx-flexspi";
```

Comment:
Is there a binding for this?

Reply:
No. Should there be? Or should I just remove the compatible?

Reply:
Ideally we'd add a binding even if there isn't a driver.

Reply:
Ok, will do